Pipfile

```diff
@@ -63,6 +62,7 @@ pytest-runner = ">=3.0.0,<5"
 # FIXME: Remove this once this issue is resolved https://github.com/pypa/pipenv/issues/3865.
 docutils = "==0.15"
 transifex-client = ">=0.12.5"
+pylint = "*"
```

Comment:
Dont include `pylintit` it is not needed by `rero-ils` and maybe someone else is using `flake`

Reply:
Done